src/fpp.h

```diff
@@ -58,6 +59,13 @@ class fpp : public cyclus::Facility  {
   /// @param time the time of the tock
   virtual void Tock();
 
+  //Resource Buffers:
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_core;
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_reserve;
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_excess;
+  cyclus::toolkit::ResBuf<cyclus::Material> helium_storage;
+  cyclus::toolkit::ResBuf<cyclus::Material> blanket;
```

Comment:
Didn't you decide that some of these didn't need to be `ResBuf`?  That said, we have new `ResBuf` capabilities, so it may be OK to keep them.

Reply:
Now that you mention it, I remember having this conversation a while back. When I go through and redo cycstub today I'll also take a look at making some of these just floating materials instead of ResBufs.